src/main/kotlin/step2/InOrderCalculator.kt

```diff
@@ -0,0 +1,39 @@
+package step2
+
+class InOrderCalculator {
+    private val ERROR_INPUT_NULL_OR_EMPTY = "입력값은 null이거나 빈 공백 문자일 수 없습니다."
+    private val ERROR_INPUT_NUMBER = "잘못된 숫자를 입력하였습니다."
+    private val ERROR_EXPRESSION = "식이 올바르지 않습니다."
+
+    private val DELIMETER = " "
+
+    fun calculate(expression: String?): Double {
+        if (expression.isNullOrBlank()) throw IllegalArgumentException(ERROR_INPUT_NULL_OR_EMPTY)
+
+        val expressions = expression.split(DELIMETER).toMutableList()
+        if (expressions.size % 2 == 0) throw IllegalArgumentException(ERROR_EXPRESSION)
+
+        var result: Double = expressions[0].toValidDouble()
```

Comment:
```suggestion
        var result: Double = expressions.firstOrNull().toValidDouble()
```
코틀린에서는 0번째 값을 first 혹은 firstOrNull이라는 함수를 제공해주고 있어요.